File: day1/Quiz3/solution.java
public class MyClass {
     public static boolean atm(String pincode){
         boolean result = false;
         int byteChk = pincode.getBytes().length ;
         boolean strChk = pincode.matches("[+-]?\\d*(\\.\\d+)?");
         if(strChk&&(byteChk == 4 || byteChk == 6)){
             result = true;
         }
         return result;
     }
    
    
    public static void main(String args[]) {
        

      System.out.println(atm("123frf24"));
    }
}

File: day1/Quiz3/solution.py
def repeatStr(i):
    result = False
    strChk = i.isdigit()
    if strChk == True and (len(i) == 4 or len(i) == 6 ):
       result = True
    print(result)

if __name__ == "__main__":
    m = "1A234"
    repeatStr(m)
